lib/manager/gradle/shallow/utils.ts

```diff
@@ -95,9 +95,19 @@ export function interpolateString(
   return resolvedSubstrings.join('');
 }
 
+export function isGradleVersionsFile(path: string): boolean {
+  const filename = upath.basename(path).toLowerCase();
+  return /^versions\.gradle(?:\.kts)?$/.test(filename);
```

Comment:
- Use invariant case flag to regex, so no need for lower case.
- Should use re2.
- Regex should be module const.

What about this?
```suggestion
  return /(^|\/|\\)versions\.gradle(?:\.kts)?$/i.test(path);
```